lua/ulx/modules/sh/cfc_arrest.lua

```diff
@@ -10,7 +10,7 @@ function cmd.arrest( callingPlayer, targetPlayers )
 
     for _, ent in ipairs( entities ) do
         local owner = ent.CPPIGetOwner and ent:CPPIGetOwner()
-        if owner and entCounts[owner] then
+        if owner then
```

Comment:
Shouldn't we check if the owner is one of the target players?

Reply:
Good point, will do.